cli/cli_app.go

```diff
@@ -56,6 +57,7 @@ GLOBAL OPTIONS:
    terragrunt-non-interactive    Assume "yes" for all prompts.
    terragrunt-working-dir        The path to the Terraform templates. Default is current directory.
    terragrunt-source             Download Terraform configurations from the specified source into a temporary folder, and run Terraform in that temporary folder
+   terragrunt-source-update      Delete the contents of the temporary folder before downloading source code into it
```

Comment:
Updated description to: "Delete the contents of the temporary folder to clear out any old, cached source code before downloading new source code into it"